packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/Controls.tsx

```diff
@@ -80,6 +82,29 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
 
   const [{ data, isLoading }, handleList] = useAction('LIST_LOCATION_ITEMS');
 
+  const [, , handleFileSelect] = useHandleUpload({
+    prefix: path,
+    preventOverwrite: true,
+  });
+  const [, handleUpdateState] = useControl('LOCATION_ACTIONS');
+
+  const handleDroppedFiles = (files: File[]) => {
+    handleFileSelect(files);
+    if (files[0].type) {
```

Comment:
what does this do?

Reply:
It is a way of determining if a file or folder was dragged, as a folder doesn't have a `type`. The purpose of that is to determine whether the action should be set to `UPLOAD_FILES` or `UPLOAD_FOLDER`

Reply:
could we have a util function for this then for readability, or a comment?